tests/integration/modules/test_cmdmod.py

```diff
@@ -383,44 +383,44 @@ def test_hide_output(self):
             'cmd.run',
             ls_command,
             hide_output=True)
-        self.assertEqual(out, '')
+        self.assertFalse(out)
 
         # cmd.shell
         out = self.run_function(
             'cmd.shell',
             ls_command,
             hide_output=True)
-        self.assertEqual(out, '')
+        self.assertFalse(out)
 
         # cmd.run_stdout
         out = self.run_function(
             'cmd.run_stdout',
             ls_command,
             hide_output=True)
-        self.assertEqual(out, '')
+        self.assertFalse(out)
 
         # cmd.run_stderr
         out = self.run_function(
             'cmd.shell',
             error_command,
             hide_output=True)
-        self.assertEqual(out, '')
+        self.assertFalse(out)
 
         # cmd.run_all (command should have produced stdout)
         out = self.run_function(
             'cmd.run_all',
             ls_command,
             hide_output=True)
-        self.assertEqual(out['stdout'], '')
```

Comment:
I think self.assertEqual(out['stderr'], '') should not be changed.
A'lot of things can be False.

Reply:
Agreed, something seems weird about this change. Why would the test be returning a bool on 2019.2.1 but a string here?

Reply:
That's true, there is a deeper problem that needs to be addressed if the same function is returning different types on different OSs.  I'll revert some of those changes and stick to the pip module test changes for now